app/server/api2/geolocation_legacy_api.py

```diff
@@ -0,0 +1,34 @@
+# third party imports
+from flask import Blueprint, make_response, jsonify
+from flask.views import MethodView
+
+# platform imports
+from server import db
+from server.models.user import User
+from server.utils.auth import requires_auth
+
+geolocation_legacy_blueprint = Blueprint('v2_geolocation_legacy', __name__)
+
+
+class GetLegacyLocation(MethodView):
```

Comment:
Will we eventually get rid of the legacy location attributes or are we maintaining both locations systems ?

Reply:
It will be replaced.

For the time being, this will be the location field editable by user (through ussd). We will need a way on knowing whether this value has been "translated" to a proper location object. At a minimum we need a "dirty" bit set somewhere to mark it. Before performing the actual migration to new location system, this has to be sorted.

I pray though that the scope of this PR doesn't have to include this; it provides framework and backend, but doesn't actually tie in with any of the currently live APIs, so it should be able to just sit there waiting to get hooked up later.